### PR TITLE
fix: match decimal points in exchange rates

### PR DESCRIPTION
**Pull request by lcsfelix**

### CODE REVIEW

**Comment by joshmh** [comment]
[Operator choice of machine-displayed crypto units](https://trello.com/c/ytcgzrUz/22-operator-choice-of-machine-displayed-crypto-units)

**Comment by naconner** [comment]
[Crypto price UI precision higher on cash-out than cash-in](https://trello.com/c/w9kFSPNN/59-crypto-price-ui-precision-higher-on-cash-out-than-cash-in)

File: ui/js/app.js [hunk 1]
```diff
@@ -1183,29 +1183,24 @@ function splitNumber(localize, localeCode) {
...
 
 function formatFiat(amount, fractionDigits) {
   if (!fractionDigits) fractionDigits = 0;
 
-  var localized = amount.toLocaleString(jsLocaleCode, {
-    useGrouping: true,
-    maximumFractionDigits: fractionDigits,
-    minimumFractionDigits: fractionDigits
-  });
-  return splitNumber(localized, jsLocaleCode) + ' ' + fiatCode;
```
Comment (RafaelTaranto) [inline ui/js/app.js]: This is used for the internationalization, we cannot remove it.

File: ui/js/app.js [hunk 2]
```diff
@@ -1183,29 +1183,24 @@ function splitNumber(localize, localeCode) {
...
-    minimumFractionDigits: 3
+    maximumFractionDigits: fractionDigits,
+    minimumFractionDigits: fractionDigits
   });
 
   return splitNumber(localized, jsLocaleCode);
 }
 
-function formatCrypto(amount) {
+function formatCrypto(amount, fractionDigits) {
```
Comment (RafaelTaranto) [inline ui/js/app.js]: This method is for formating crypto values, I don't think we need to change it

File: ui/js/app.js [hunk 3]
```diff
@@ -1223,7 +1218,7 @@ function setExchangeRate(_rates) {
 
   if (rates.cashOut) {
     var cashOut = new BigNumber(rates.cashOut);
-    var cashOutCryptoToFiat = cashOut && formatCrypto(cashOut.round(3).toNumber());
+    var cashOutCryptoToFiat = cashOut && formatCrypto(cashOut.round(3).toNumber(), 2);
```
Comment (RafaelTaranto) [inline ui/js/app.js]: See on line 1216 how we do it for the cash-in, we should the same here. I.E. replace formatCrypto with formatFiat and add the second parameter "2".

File: ui/js/app.js [hunk 4]
```diff
@@ -1183,29 +1183,24 @@ function splitNumber(localize, localeCode) {
   return ['<span class="integer">', split[0], '</span><span class="decimal-char">', decimalChar, '</span><span class="decimal">', split[1], '</span>'].join('');
 }
 
-function formatNumber(num) {
+function formatNumber(num, fractionDigits = 3) {
```
Comment (RafaelTaranto) [inline ui/js/app.js]: No changes needed on this function, since it's only used for formatting crypto.

File: ui/js/app.js (hunk 3 above)
Comment (lcsfelix) [reply ui/js/app.js]: Changing only this results still results in 3 decimal points, with a trailing zero, due to the way the formatNumber, which is called by formatCrypto, is written

**Pull request closed without merging**